src/System.CommandLine.Suggest.Tests/TestsWithTestApps.cs

```diff
@@ -0,0 +1,74 @@
+using System.IO;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.CommandLine.Suggest.Tests;
+
+[Collection("TestsWithTestApps")]
```

Comment:
Note: this attribute is inherited, so both SuggestionStoreTests and DotnetSuggestEndToEndTests will run in the same collection, meaning - not in parallel. This allows the gracefull cleanup of TestsApp files.